dll/shellext/zipfldr/CConfirmReplace.cpp

```diff
@@ -3,17 +3,17 @@
  * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
  * PURPOSE:     Ask the user to replace a file
  * COPYRIGHT:   Copyright 2017-2019 Mark Jansen (mark.jansen@reactos.org)
+ *              Copyright 2023 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
  */
 
 #include "precomp.h"
 
 class CConfirmReplace : public CDialogImpl<CConfirmReplace>
 {
 private:
-    CStringA m_Filename;
+    CStringW m_Filename;
 public:
-
-    CConfirmReplace(const char* filename)
+    CConfirmReplace(LPCWSTR filename)
```

Comment:
`LPCWSTR` --> `PCWSTR` (and if any, `LPWSTR` -> `PWSTR`) here and in your other changes.

Reply:
Replaced them.

Reply:
@HBelusca what is the point of this?

Reply:
The prefix `LP` means FAR pointer.
The prefix `P` means NEAR pointer.
`LP` is provided for compatibility to Win16.
On Win32 and Win64, the prefixes `LP` and `P` have the same type. No need to distinguish them.

@HBelusca just wants unification for consistency.